packages/pds-ember/addon/components/pds/app/drawer/index.hbs

```diff
@@ -0,0 +1,9 @@
+<div
+  class="pds-app__drawer pds-drawer"
+  role="dialog"
+  aria-expanded="{{if @open 'true' 'false'}}"
```

Comment:
The `@open` argument is set automatically by `Pds::App` if you use the contextual `<App.Drawer>` component like so:

_closed_
```handlebars
<Pds::App as |App|>
  <App.Drawer>...</App.Drawer>
</Pds::App>
```

_open_
```handlebars
<Pds::App @showDrawer={{true}} as |App|>
  {{!-- Drawer already has @open={{true}} defined by Pds::App --}}
  <App.Drawer>...</App.Drawer>
</Pds::App>
```